test/Parse/line-directive.swift

```diff
@@ -14,9 +14,17 @@ x // expected-error {{parameterless closing #sourceLocation() directive without
 
 #sourceLocation(file: x.swift, line: 1) // expected-error{{expected filename string literal}}
 
-// expected-warning@+1 {{expected starting line number}}
+// expected-warning@+1 {{expected starting line number for #sourceLocation directive; this is an error in Swift 6}}
 #sourceLocation(file: "x.swift", line: 0xff)
 
+#sourceLocation()
+
+// expected-warning@+1 {{'#sourceLocation' cannot be a multiline string literal; this is an error in Swift 6}}
+#sourceLocation(file: """
+x.swift
+y.swift
+""", line: 42)
```

Comment:
For e.g `#sourceLocation(file: "a\nb", line: 42)`, seems we just take the characters literally, not sure if we ought to ban escape sequences too to avoid confusion.

Reply:
Hmm, interesting. Let’s consider that a separate issue though. It’s probably not worth fixing it in the old parser anyway.